### PR TITLE
OSSM-3709 Fix nightly builds and add publish nightly bundle

### PR DESCRIPTION
**Pull request by dgn**

### CODE REVIEW

**Review by yxun — APPROVED** [review]

**Comment by dgn** [comment]
/pj-rehearse periodic-ci-maistra-istio-operator-maistra-3.0-nightly-images-3-0

File: ci-operator/config/maistra/istio-operator/maistra-istio-operator-maistra-3.0.yaml [hunk 1]
```diff
@@ -139,25 +139,26 @@ tests:
...
-          export REPO=quay.io/maistra-dev/istio-ubi9-operator; \
-          export IMAGE=$REPO:3.0-latest; \
-          make -e COMMUNITY=false clean image; \
-          docker tag $IMAGE $REPO:3.0-nightly-`date -I`; \
-          docker push $IMAGE; \
-          docker push $REPO:3.0-nightly-`date -I`'
+          sh -c \
+          "git config --global user.email 'dgrimm@redhat.com'; \
+           git config --global user.name 'Daniel Grimm'; \
+           make -e GITHUB_TOKEN=${GITHUB_TOKEN} docker-push-nightly bundle-publish-nightly"
```
Comment (yxun) [inline ci-operator/config/maistra/istio-operator/maistra-istio-operator-maistra-3.0.yaml:153]: Does this mean the nightly job will create a PR to operator community repo nightly ?
I can approve this PR and then we can update the script from operator repo is needed.

Comment (dgn) [reply ci-operator/config/maistra/istio-operator/maistra-istio-operator-maistra-3.0.yaml:153]: yep, it creates a PR every night

**Pull request merged**